### DISCUSSION TITLE
net6.0 compatibility

### DISCUSSION THREAD

**Discussion by chanakya-svt** [JosefPihrt/Roslynator#868] [General]
Using Roslynator CLI v0.3.0 with .net v6 and getting the following error when running 

`roslynator analyze` or `fix` or `format` commands.

Are there any extra configuration steps for CLI to work with .net v6?

```
It was not possible to find any compatible framework version
The framework 'Microsoft.NETCore.App', version '5.0.0' (x64) was not found.
  - The following frameworks were found:
      3.1.21 at [/usr/local/share/dotnet/shared/Microsoft.NETCore.App]
      6.0.0 at [/usr/local/share/dotnet/shared/Microsoft.NETCore.App]

You can resolve the problem by installing the specified framework and/or SDK.

The specified framework can be found at:
  - https://aka.ms/dotnet-core-applaunch?framework=Microsoft.NETCore.App&framework_version=5.0.0&arch=x64&rid=osx.11.1-x64
```

**Note**: All the above commands are working with .net v3.1

**Comment by josefpihrt** [comment]
Hi,

Currently, Roslynator command-line tool requires .NET Core SDK 5.0:

https://github.com/JosefPihrt/Roslynator/blob/master/docs/cli/README.md

> Version 0.2.0 or higher requires .NET Core SDK 5.0


Future release could support both 5.0 and 6.0. Would you mind to create an issue for it?
